tests/jest.config.unit.ts

```diff
@@ -84,6 +85,7 @@ export default {
 		'tsx',
 		'json',
 		'node',
+		'mts',
```

Comment:
What is this mts?

Reply:
oh, its a way to get jest config to work, but thanks, I don't need it, will remove t.

Reply:
removed